packages/common/src/search/_internal/hubSearchEvents.ts

```diff
@@ -9,7 +9,22 @@ import { processOptions } from "./hubEventsHelpers/processOptions";
 import { processFilters } from "./hubEventsHelpers/processFilters";
 
 /**
- * Searches for events against the Events 3 API using the given `query` and `options`
+ * Searches for events against the Events 3 API using the given `query` and `options`.
+ * Currently supported filters include:
+ *   access: string;
+ *   canEdit: boolean
+ *   entityIds: string;
+ *   entityTypes: string;
+ *   eventIds: string;
+ *   term: string;
+ *   categories: string;
+ *   tags: string;
+ *   readGroupIds: string;
+ *   editGroupIds: string;
+ *   attendanceType: string;
+ *   owner: string;
+ *   status: string;
```

Comment:
Upon thinking about these docs a bit more, some of them expect explicit string values, may be better the explicitly define the expected values. Per above comment, lets update some of these to be singular too. I think the following should work:

```ts
/**
 * Searches for events against the Events 3 API using the given `query` and `options`.
 * Currently supported filters include:
 *   access: 'public' | 'private' | 'org';
 *   canEdit: boolean
 *   entityId: string;
 *   entityType: string;
 *   eventId: string;
 *   term: string;
 *   categories: string;
 *   tags: string;
 *   readGroupId: string;
 *   editGroupId: string;
 *   attendanceType: 'virtual' | 'in_person';
 *   owner: string;
 *   status: 'planned' | 'canceled' | 'removed';
 *   startDateRange: IDateRange<string | number>
 * @param query An IQuery object
 * @param options An IHubSearchOptions object
 * @returns a promise that resolves a <IHubSearchResponse<IHubSearchResult> object
 */
```